package.json

```diff
@@ -10,6 +10,10 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
+    "prettier": "3.0.3",
     "typescript": "^5.2.2"
+  },
+  "dependencies": {
+    "@types/node": "^20.5.9"
```

Comment:
This should be in `devDependencies` because types are only needed at compile time, not run time.